src/test/java/org/encorapedia/EncorapediaReportPricesResultsTest.java

```diff
@@ -0,0 +1,84 @@
+package org.encorapedia;
+
+import com.sun.istack.internal.NotNull;
+import org.browser.BrowserFactory;
+import org.encorapedia.pages.EncorapediaHomePage;
+import org.encorapedia.pages.EncorapediaReportsPage;
+import org.encorapedia.pages.EncorapediaResultsPage;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+
+import java.time.LocalDate;
+import java.time.ZoneId;
+import java.util.Date;
+import java.util.List;
+
+public class EncorapediaReportPricesResultsTest {
+    private WebDriver driver;
+
+    public void chrome() throws Exception {
+        driver= BrowserFactory.getBrowser("chrome");
+        driver.manage().window().maximize();
+        driver.get("http://localhost:8080/choice-selenium-testing/index.html");
+    }
+
+    @Before
```

Comment:
This is a personal recommendation not mandatory. I like to organize my test classes in one of the following ways:
**One**
Constants
Class attributes
BeforeSuite
BeforeClass
BeforeTest
AfterTest
AfterClass
AfterSuite
Tests
private methods

**Two**
Constants
Class attributes
BeforeSuite
BeforeClass
BeforeTest
Tests
AfterTest
AfterClass
AfterSuite
private methods

As I mentioned, this os something that works for me.

Reply:
That's a good way to organize it. I'll take your recommendation and I'm going to start using that structure. Thanks!